identity-service/src/notifications/index.js

```diff
@@ -1221,19 +1226,39 @@ class NotificationProcessor {
         emailParams['subject'] = 'Unread notifications from last week'
       }
 
-      console.log(emailParams)
       await this.sendEmail(emailParams)
+
       // Temporary debugging email
       let emailParams2 = emailParams
       emailParams2['to'] = 'hareesh@audius.co'
+      emailParams2['bcc'] = 'forrest@audius.co'
       await this.sendEmail(emailParams2)
+
+      // Cache on file system
+      await this.cacheEmail({ renderProps, emailParams })
+
       return true
     } catch (e) {
       console.log(`Error in renderAndSendEmail ${e}`)
       return false
     }
   }
 
+  async cacheEmail (cacheParams) {
+    let uuid = uuidv4()
+    let timestamp = moment().valueOf()
+    let fileName = `${uuid}-${timestamp.toString()}.json`
+    let filePath = path.join(emailCachePath, fileName)
+    await new Promise((resolve, reject) => {
+      fs.writeFile(filePath, JSON.stringify(cacheParams), (err) => {
+        if (err) {
+          console.log(err)
```

Comment:
should this be a reject maybe?

Reply:
i think just resolve is fine. this should never trigger the catch clause in the renderAndSendEmail function , just logging is probably sufficient